### DISCUSSION TITLE
MySqlBulkCopy & EF Core value converters

### DISCUSSION THREAD

**Discussion by DisplayL** [mysql-net/MySqlConnector#1288] [Q&A] [answered]
Hi everyone, I can't find this in the internet so I decided to post this question here. MySqlBulkCopy doesn't support the value converters of EF Core?

**Comment by bgrainger** [comment, marked as answer]
Correct; the only "converters" `MySqlBulkCopy` supports are expressions that you set in `MySqlBulkCopyColumnMapping`: https://mysqlconnector.net/api/mysqlconnector/mysqlbulkcopy/columnmappings/.
